client/js/libs/handlebars/parse.js

```diff
@@ -16,11 +16,20 @@ function createFragment(fragment) {
 	if (fragment.bold) {
 		classes.push("irc-bold");
 	}
-	if (fragment.textColor !== undefined) {
-		classes.push("irc-fg" + fragment.textColor);
-	}
-	if (fragment.bgColor !== undefined) {
-		classes.push("irc-bg" + fragment.bgColor);
+	if (fragment.reverse) {
```

Comment:
I think it's better to implement these in `parseStyle` so the fragments can be optimized correctly.

Reply:
Happy to do this, but if we implement these in `parseStyle` doesn't that render the `reverse` flag in the fragment moot? Namely all logic/testing can be done via the `textColor` and `bgColor` and `reverse` seems somewhat redundant

Reply:
> doesn't that render the reverse flag in the fragment moot?

I believe so, yeah. Feel free to remove it.